alpaca/rest.go

```diff
@@ -1177,3 +1178,29 @@ func unmarshal(resp *http.Response, data interface{}) error {
 
 	return json.Unmarshal(body, data)
 }
+
+type localPlaceOrderRequest PlaceOrderRequest
+
+func (req PlaceOrderRequest) MarshalJSON() ([]byte, error) {
```

Comment:
This is very hacky, but I couldn't find a better solution either with the standard json lib. Let's go with it :D

Reply:
Does the SDK do any field validation check? If not, then just pass the request as is to the API and let it return?

But if you want some validations on client side, then just check if only one of qty or notional is non-zero? Ideally we could make them both pointers so we could do a more explicit nil check but that would be a breaking change.

Reply:
The only reason for this hack is to maintain backward compatibility in the exported structs.